addons/osfstorage/models.py

```diff
@@ -413,13 +416,39 @@ def serialize(self, include_full=False, version=None):
         return ret
 
 
+class Region(models.Model):
+    _id = models.CharField(max_length=255, db_index=True)
+    name = models.CharField(max_length=200)
+    waterbutler_credentials = DateTimeAwareJSONField(default=dict)
```

Comment:
This field should be encrypted before it goes in the database, since there's sensitive info in here. It might makes sense to create a subclass of `JSONField` that transparently encrypts/decrypts values (like our EncryptedTextField). 

h/t @felliott for catching this.